src/index.ts

```diff
@@ -124,3 +126,45 @@ async function _replacePlaceholder(wdioConf: string) {
     fileString = fileString.replace(/%baseUrl%/g, BASE_URL)
     await fs.writeFile(wdioConf, fileString)
 }
+
+function _getUsedPackageManager(): PackageManager {
+    const alternativeManagers = [PackageManager.pnpm, PackageManager.pnpm]
+    const defaultManager = PackageManager.npm
+
+    const { npm_config_user_agent } = process.env
```

Comment:
there is a `user agent` also for package managers, wow. cool, learned something new here!

Reply:
Yeah, this was my first go-to, just logging process.env to see if there is something interesting. Fortunately there is :)